makefiles/boards/sam0.inc.mk

```diff
@@ -18,6 +18,11 @@ endif
 # Default for these boards is to use a CMSIS-DAP programmer
 DEBUG_ADAPTER ?= dap
 
+ifeq (dfu-util,$(PROGRAMMER))
+  DFU_USB_ID ?= ${USB_VID}:${USB_PID}
```

Comment:
Somehow this is not working when flashing the nrf52840dk:
```
koen@morgen ~/dev/RIOT-review $ USB_VID=1209 USB_PID=7d01 DFU_ALT=0 USEMODULE=usbus_dfu PROGRAMMER=dfu-util BOARD=nrf52840dk make -C  examples/gnrc_networking riotboot/flash-slot0          
make: Entering directory '/home/koen/dev/RIOT-review/examples/gnrc_networking'
compiling /home/koen/dev/RIOT-review/dist/tools/riotboot_gen_hdr/bin/genhdr...
make: Nothing to be done for 'all'.
...Compile Compile...
creating /home/koen/dev/RIOT-review/examples/gnrc_networking/bin/nrf52840dk/gnrc_networking-slot0.1607449046.riot.bin...
dfu-util --device  --alt 0 --download /home/koen/dev/RIOT-review/examples/gnrc_networking/bin/nrf52840dk/gnrc_networking-slot0.1607449046.riot.bin 
dfu-util 0.9
```

Reply:
I'll have a look.

Reply:
Fixed.